cli/cmd/check.go

```diff
@@ -143,7 +143,10 @@ func configureAndRunChecks(cmd *cobra.Command, wout io.Writer, werr io.Writer, o
 	}
 
 	crdManifest := bytes.Buffer{}
-	err = renderCRDs(&crdManifest, valuespkg.Options{}, "yaml")
+	err = renderCRDs(&crdManifest, valuespkg.Options{
+		// GatewayAPI CRDs are optional so don't check for them.
+		Values: []string{"enableHttpRoutes=false"},
+	}, "yaml")
```

Comment:
This seems like a separate change Unrelated to the policy controller. Can we split it out?

Reply:
We could, but I consider both of these to be part of the same feature -- that Linkerd should work when the GrpcRoute CRD is absent.